TrafficCapture/testingEnvironment/build.gradle

```diff
@@ -0,0 +1,125 @@
+plugins {
+    id 'org.opensearch.migrations.java-library-conventions'
+    id "com.avast.gradle.docker-compose" version "0.16.12"
+    id "com.bmuschko.docker-java-application" version "9.3.1"
+}
+
+import java.security.MessageDigest
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+import org.apache.tools.ant.taskdefs.condition.Os
+
+def calculateDockerHash(String projectName) {
+    MessageDigest digest = MessageDigest.getInstance('SHA-256')
+    fileTree("src/main/docker/${projectName}")
+            .each( file ->
+                    file.withInputStream { is ->
+                        var buffer = new byte[1024]
+                        int read
+                        while ((read = is.read(buffer)) != -1) {
+                            digest.update(buffer, 0, read)
+                        }
+                    }
+            )
+    return digest.digest().encodeHex().toString()
+}
+
+dependencies {
+    implementation project(':trafficCaptureProxyServer')
+    testImplementation project(':trafficCaptureProxyServerTest')
+}
```

Comment:
This can fall away by merging these changes into the ...ProxyServerTest directory